infra/enrolment-records-store/src/main/java/com/simprints/infra/enrolment/records/store/EnrolmentRecordRepositoryImpl.kt

```diff
@@ -109,4 +115,30 @@ internal class EnrolmentRecordRepositoryImpl(
             project = project
         )
     }
+
+    override suspend fun count(query: SubjectQuery, dataSource: BiometricDataSource): Int =
```

Comment:
Does it still make sense to have the delegation in the declaration? Should the repo even implement the local data source interface in the first place?

Reply:
On the first question - do you mean to move the current implementation from the repo to the interface?
On the second - it doesn't need it actually. Will remove.

Reply:
I know it does, but should it?

Reply:
Not sure I get you - the local data source offers more functions as we also modify it unlike the CommCare one which we just read. So, in that regard we do need the extra functionality from the local data source.